docs/data-structures/data-structures.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: What are Data Structures?
+slug: /data-structures
+---
+
+## Introduction
+
+A program is a set of instructions that performs operations on data. The way of organizing this data in the main memory at the time of execution of the program is called ***data structures*** .
+
+Depending on the requirement of a program and the type of procedures it is performing, we have various data structures. Some of the common examples of data structures include arrays, linked lists, stacks,  queues, trees and graphs. Data structures
+are widely applied in the following areas: 
+* Compiler design  
+* Operating system
+* DBMS
+* Simulation
+* Artificial intelligence 
+* Graphics
+
+## Classification of Data Structures
+
+Data structures are classified into two classes: 
+
+#### ***Primitive and Non-primitive Data Structures*** 
+
+Primitive data structures are the basic data types of any programming
+language that form the basic unit for the data structures. Some basic data types are integer, float, character and boolean.           
+    Non-primitive data structures are complex structures that are created using primitive data structures. Examples of such data structures include Arrays, Linked Lists, Stacks, Queues, Trees, and Graphs.
+
+![Flowchart](https://i.imgur.com/0hLAz15.png) 
```

Comment:
Have you made the diagram yourself?

Reply:
Yes I made it using Creately app. Here's the [link](https://app.creately.com/diagram/HCKrmS4ayh6/view) for the same.